src/main/java/lotto/domain/LottoGame.java

```diff
@@ -19,29 +13,9 @@ public LottoTickets getTickets() {
         return tickets;
     }
 
-
-    private Map<Rank, Integer> initializeResults() {
-        // Rank의 모든 값을 초기화하여 결과 맵 생성
-        Map<Rank, Integer> results = new EnumMap<>(Rank.class);
-        for (Rank rank : Rank.values()) {
-            results.put(rank, 0);
-        }
-        return results;
-    }
-
-    public GameResult gameStart(List<Integer> winningNumbers) {
-        Map<Rank, Integer> results = initializeResults();
-
-        // 각 티켓의 당첨 등수를 계산하여 결과 맵에 반영
-        tickets.getLottoTickets().stream()
-                .map(ticket -> Rank.getRankByMatchCount(ticket.matchLottoNumbers(winningNumbers)))
-                .forEach(rank -> results.merge(rank, 1, Integer::sum));
-
-        // 수익률 계산
-        double returnRate = calculateReturnRate(results);
-
-        // 결과 객체 생성 및 반환
-        return new GameResult(results, returnRate);
+    public GameResult gameStart(LottoTicket winningTicket) {
+        Map<Rank, Integer> results = tickets.match(winningTicket);
+        return new GameResult(results, calculateReturnRate(results));
```

Comment:
피드백 반영 👍 
수익률 계산이라는 기능도 객체에게 위임해 보시면 어떨까요?
```suggestion
        MatchResult results = tickets.match(winningTicket);
        double returnRate = results.calculateRate(구입금액);
        return new GameResult(results, returnRate);
```